### ISSUE TITLE
Support for Outposts CLI

### ISSUE THREAD

**Issue by perrwald** [donnemartin/saws#121]
Currently Outposts CLI, and outposts related subsections within general CLI are not supported. Can they be added?

**Comment by dhirensr** [comment]
good point, I guess this needs to be added in saws : https://docs.aws.amazon.com/cli/latest/reference/outposts/
is anyone looking at it? otherwise I can try to add it.
